utilities/logger.py

```
@@ -27,7 +27,7 @@ def filter(self, record):
 
 
 class TestLogFormatter(ColoredFormatter):
-    def formatTime(self, record, datefmt=None):  # noqa: N802
+    def formatTime(self, record, datefmt=None):
```

Ruff (0.15.12)

[warning] Unused method argument: `datefmt`

(ARG002)